app/models/manageiq/providers/google/inventory/collector.rb

```diff
@@ -34,8 +34,7 @@ def cloud_database_flavors
   end
 
   def flavors
-    flavors_by_zone = compute.list_aggregated_machine_types.items
-    flavors_by_zone.values.flat_map(&:machine_types).compact.uniq(&:id)
+    compute.machine_types.all
```

Comment:
TODO I want to do extra validation that this is the same thing, but this code suffers from the same missing nextPageToken handling as #all did

Reply:
Okay I ran a refresh with and without this change and serialized the entire flavors table using `Spec::Support::EmsRefreshHelper` and the tables are identical
```
>> flavors = serialize_inventory([Flavor]).to_json
>> File.write("flavors_before.json", flavors)
```
```
~/src/manageiq/manageiq$ ls -l flavors_*
-rw-r--r-- 1 adam grare 124699 Oct 26 09:17 flavors_after.json
-rw-r--r-- 1 adam grare 124699 Oct 26 09:24 flavors_before.json
~/src/manageiq/manageiq$ diff flavors_before.json flavors_after.json 
:~/src/manageiq/manageiq$ 
```